runhouse/resources/hardware/cluster.py

```diff
@@ -1946,3 +1954,65 @@ def _folder_mv(
 
     def _folder_exists(self, path: Union[str, Path]):
         return self.client.folder_exists(path=path)
+
+    ###############################
+    # Cluster list
+    ###############################
+    @classmethod
+    def list(
+        cls,
+        show_all: Optional[bool] = False,
+        since: Optional[str] = None,
+        status: Optional[Union[str, ResourceServerStatus]] = None,
+    ):
+        """
+        Returns user's runhouse clusters saved in Den. If filters are provided, only clusters that are matching the
+        filters are returned. If not filters are provided, clusters that were active in the last 24 hours are returned.
+
+        Args:
+            show_all (bool, optional): Whether to list all clusters saved in Den (max 50).
+            since (str, optional): Clusters that were active in the specified time period will be returned. Value can be in seconds (65s), minutes (3m), hours (2h) or days (7d).
```

Comment:
```suggestion
            since (str, optional): Clusters that were active in the specified time period will be returned. Value can be in seconds, minutes, hours or days.
```

I think would be good to show some examples in the docstrings below for how to use the "since" param:

```
    Examples:
        >>> Cluster.list(since="15s")
        >>> Cluster.list(since="3m")
        >>> Cluster.list(since="2h")
        >>> Cluster.list(since="7d")
```